p2p/host/autorelay/autorelay.go

```diff
@@ -135,6 +138,75 @@ func (ar *AutoRelay) background(ctx context.Context) {
 	}
 }
 
+func (ar *AutoRelay) refresh(ctx context.Context) {
+	ticker := time.NewTicker(time.Minute)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			var toRefresh []peer.ID
+
+			ar.mx.Lock()
+			if ar.status == network.ReachabilityPublic {
+				// we are public, forget about the relays, unprotect peers
+				for p := range ar.relays {
+					ar.host.ConnManager().Unprotect(p, "autorelay")
```

Comment:
Nit: The string should be a package-level `const`.

Reply:
sure.

Reply:
done.